src/main/java/fi/helsinki/cs/tmc/core/spyware/DiffSender.java

```diff
@@ -55,7 +55,6 @@ public HttpResult sendToUrl(byte[] diffs, String url) {
             return urlCommunicator.makePostWithByteArray(
                             url, diffs, createHeaders(), new HashMap<String, String>());
         } catch (IOException ex) {
-            System.err.println(ex.getMessage());
             return new HttpResult(ex.getMessage(), 500, false);
```

Comment:
We maybe should also log this -- and make sure we treat the snapshot data correctly